adapters/imds/imds.go

```diff
@@ -14,7 +14,7 @@ import (
 	"github.com/prebid/prebid-server/openrtb_ext"
 )
 
-type SynacorMediaAdapter struct {
+type ImdsAdapter struct {
```

Comment:
Please name this type `adapter`. There is no need to repeat the namespace name. There is no need to export the type by starting with an uppercase character. It's likely the SynacorMedia adapter predated this advice.